### PR TITLE
Preferences UI tweaks

### PR DESCRIPTION
**Pull request by ronso0**
Just a few commits off my stash list. See commits for details.

7b83ac784bf465c3984af7b17bc1eb590e5cc1db contains a new tr string, let me know if we should rather merge that to main.

### CODE REVIEW

**Comment by Holzhaus** [comment]
> 7b83ac7 contains a new tr string, let me know if we should rather merge that to main.

Just wondering, when was the last time we pushed translatable strings to transifex? We probably should do that soon to avoid ending up with a bunch of untranslated stuff when we release 2. 4.

**Comment by daschuer** [comment]
I have just updated the translations.

**Comment by daschuer** [comment]
If the string change is not fixing wrong information, it is better to move it to 2.5. New strings can be added at anytime.

**Comment by ronso0** [comment]
FYI it's no important UI string, just a tooltip for music directories in the Library preferences that are currently not available / invalid.
I don't mind moving the commit.

**Comment by daschuer** [comment]
I have no special demands here, both works for me.

**Review by daschuer — APPROVED** [review]
Works good and looks good. The string is helpful even if it sticks untranslated. 
Thank you.

File: src/preferences/dialog/dlgprefinterface.cpp [hunk 1]
```diff
@@ -125,12 +125,7 @@ DlgPrefInterface::DlgPrefInterface(
 
     if (pSkinLoader) {
         // Skin configurations
-        QString sizeWarningString =
-                "<img src=\":/images/preferences/ic_preferences_warning.svg\") "
-                "width=16 height=16 />   " +
-                tr("The minimum size of the selected skin is bigger than your "
-                   "screen resolution.");
-        warningLabel->setText(sizeWarningString);
+        warningLabel->setText(kWarningIconHtmlString);
```
[the remark this answers is not in the document]
Comment (ronso0) [reply src/preferences/dialog/dlgprefinterface.cpp:128]: I'll prepare a fixup asap

[the remark this answers is not in the document]
Comment (ronso0) [reply src/preferences/dialog/dlgprefinterface.cpp:128]: Uff, damn, hopefully the translations have not been dumped, yet.
@daschuer (or whoever is going to push tr) please wait for the fixup!

**Pull request merged**